purchase.adoc

```diff
@@ -375,3 +404,97 @@ purchaseUUID:: The UUID of the purchase
     "refunded": false
 }
 ```
+
+### Discounts
+Discounts can exist both on a specific row of items in the shopping cart, as well as on the entire shopping cart. A discount can be set either as a percentage or as a fixed value.
+
+When applying the total discounts on the purchase price, the discounts set on rows will be calculated first and subtracted from the total price followed by any discount set on the entire chart.
+
+Row discounts are expressed using the following JSON structure in the `products` list, containing either `amount` for fixed discounts or `percentage` for percentage discounts:
+```json
+"discount": {
+    "quantity": 1,
```

Comment:
I am left wondering if the quantity is always 1 on the row discount. What if I discount apples with 1kr for certain customers and one of those customers buys 5 apples. How is that expressed?